### ISSUE TITLE
InvalidOperationException: Aql functions should only be used in ArangoDatabase.Query

### ISSUE THREAD

**Issue by drgraduss** [ra0o0f/arangoclient.net#28]
Every time you're using AQL within query ```InvalidOperationException: Aql functions should only be used in ArangoDatabase.Query``` exception thrown and caught. This affects query construction performance.

Adding this for tracking.

**Comment by ra0o0f** [comment]
@drgraduss thanks for open an separate issue for this. i'm aware of the problem.

**Comment by drgraduss** [comment]
Hi @ra0o0f. When to expect fix for this issue? Thanks

**Comment by ra0o0f** [comment]
@drgraduss not until two month

**Comment by ra0o0f** [comment]
@drgraduss this should be and will be fixed. but compare these:

```c#
Stopwatch watch = new Stopwatch();
                watch.Start();

                for(int i=0;i<10000;i++)
                    db.Query<Person>()
                        .Where(p=> AQL.Contains(p.Fullname,"raoof"))
                        .Select(p => p).GetQueryData();

                watch.Stop();
                Console.WriteLine(watch.Elapsed);
```

```c#
Stopwatch watch = new Stopwatch();
                watch.Start();

                for(int i=0;i<10000;i++)
                    db.Query<Person>()
                        .Where(p=>p.Fullname == "raoof")
                        .Select(p => p).GetQueryData();

                watch.Stop();
                Console.WriteLine(watch.Elapsed);
```
